create_vesting.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# fail immediately if a command fails
+set -eo pipefail
+
+if [[ -z ${1} ]]; then
+	echo "First argument (vester) not found, please set it and re-run the last command."
+	exit 1
+fi
+VESTER=$1
+
+if [[ -z ${2} ]]; then
+	echo "Second argument (signer) not found, please set it and re-run the last command."
+	exit 2
```

Comment:
1 for general errors

```suggestion
	exit 1
```